.teamcity/src/main/kotlin/configurations/FlakyTestQuarantine.kt

```diff
@@ -53,7 +53,7 @@ class FlakyTestQuarantineProject(
         model.stages
             .filter { it.stageName <= StageName.READY_FOR_RELEASE }
             .flatMap { it.functionalTests }
-            .filter { it.os == os }
+            .filter { it.os == os && !it.testType.crossVersionTests }
```

Comment:
❓Could they run with Test Distribution instead?

Reply:
These tests on Linux ran successfully but failed with "too many artifact errror": https://builds.gradle.org/buildConfiguration/Gradle_Master_Check_FlakyQuarantine_Check_AllVersionsCrossVersion_10/104308173?buildTab=overview&hideTestsFromDependencies=true

These tests on Windows couldn't finish because of timeout: https://builds.gradle.org/buildConfiguration/Gradle_Master_Check_FlakyQuarantine_Check_AllVersionsCrossVersion_11/104308161?buildTab=overview&hideTestsFromDependencies=true